connectors/es/sink.py

```diff
@@ -569,40 +569,29 @@ async def get_access_control_docs(self, generator):
                 self._log_progress()
 
             doc_id = doc["id"] = doc.pop("_id")
-            doc_exists = doc_id in existing_ids
```

Comment:
This is where it would be good if our List Roles API could find roles specific to this connector. 🤔